lib/bel/translator/plugins/rdf/translator.rb

```diff
@@ -94,6 +94,16 @@ def write_rdf_to_io(format, objects, io, options = {})
         :stream => true
       )
 
+      if (@format == :turtle)
+        require 'yaml'
+
+        prefixes = YAML::load_file('config/namespace_prefixes.yml')
```

Comment:
Resolved using the Pathname class: Pathname.new('config/namespace_prefixes.yml')